deploy/contents/install/app/install-utils.sh

```diff
@@ -1004,7 +1017,12 @@ function get_kube_resource_spec_file_by_type {
         local spec_suffix="${CP_KUBE_SERVICES_TYPE:-"node-port"}"
         
         echo "${spec_dir}/${spec_file}-${spec_suffix}.${spec_ext}"
-    else
+    
+    elif [ "$resource_type" == "--ktz" ]; then 
+        template_file="/tmp/csi_${RANDOM}.yaml"
```

Comment:
why you are using `csi` preix, under kustomize config could be any type of the resource, so let's use something more common like `cp_kube_res_` 
and also let's also add to the name of the temporary res file name of the kustomize directory:
```
local spec_dir="$(basename $original_spec_file)"
template_file="/tmp/cp_kube_res_${spec_dir}_${RANDOM}.yaml"
```

Reply:
Also it is better to change the name of the variable `original_spec_file` to `original_spec_location` (similar to function `create_kube_resource`) since now it can be a folder an not a file